VERSION

```diff
@@ -0,0 +1 @@
+v0.3.0
```

Comment:
This should not include `v`. `v` should only be used when printing the version via `$ pick version`